File: resources/expected/ros2/isolette_inverted_topics/lax/src/isolette_cpp_pkg/src/base_code/heat_source_cpi_heat_controller_runner.cpp
#include "isolette_cpp_pkg/user_headers/heat_source_cpi_heat_controller_src.hpp"

//========================================================
// Re-running Codegen will overwrite changes to this file
//========================================================

heat_source_cpi_heat_controller::heat_source_cpi_heat_controller() : heat_source_cpi_heat_controller_base()
{
    // Invoke initialize entry point
    initialize();

    PRINT_INFO("heat_source_cpi_heat_controller infrastructure set up");
}

int main(int argc, char **argv)
{
    rclcpp::init(argc, argv);
    auto executor = rclcpp::executors::MultiThreadedExecutor();
    auto node = std::make_shared<heat_source_cpi_heat_controller>();
    executor.add_node(node);
    executor.spin();
    rclcpp::shutdown();
    return 0;
}

File: resources/expected/ros2/python-isolette/strict/src/isolette_py_pkg/isolette_py_pkg/base_code/heat_source_cpi_heat_controller_runner.py
#!/usr/bin/env python3
import rclpy
from rclpy.node import Node
from rclpy.executors import MultiThreadedExecutor
from isolette_py_pkg.user_code.heat_source_cpi_heat_controller_src import heat_source_cpi_heat_controller
#========================================================
# Re-running Codegen will overwrite changes to this file
#========================================================
def main(args=None):
    rclpy.init(args=args)
    node = heat_source_cpi_heat_controller()
    executor = MultiThreadedExecutor()
    executor.add_node(node)
    executor.spin()
    rclpy.shutdown()

if __name__ == "__main__":
    main()
